### ISSUE TITLE
Very low volume

### ISSUE THREAD

**Issue by rastapopougros** [masmu/pulseaudio-dlna#144]
I try to use DLNA with ubuntu, rhythmbox, and my new hifi (Teac CR-H700).

Start pulseaudio-dlna with MP3 codec : ok.
Discover my hifi device : ok.
Choose my hifi device on ubuntu audio settings (or pavu) : ok.

My hifi find that a DLNA stream is sent and display the right things (like time since begining and  "livestream from `<my computer>`").

BUT : I don't hear anything, so I boost the volume and I have to go to 45 (of 50 !) to hear a poor bad quality sound.
Protip : normally at 20-25 of 50 it is already loud… so I think it's normal that the quality is bad at 45.

So the problem is that the volume which come into my hifi by the home wifi seems to be very very low.

(And yes, all is at 100% in audio settings on pavu.)

**Comment by masmu** [comment]
Well, there are basically 3 audio levels.
- The audio volume of the audio file itself.
- The master audio volume (found in pavucontrol on the 3rd tab: output devices)
- The stream audio volume (found in pavucontrol on the 1st tab: playback)

Is there a chance that you missed the master audio volume?

**Comment by rastapopougros** [comment]
Sorry for the delay,

but… like I said previously : "everything is at 100% in audio settings or in pavu"

Playback tab / Rhythmbox : 100%
Recording tab / "parec" : 100%
Output tab / My device (DLNA) : 100%

**Comment by towo** [comment]
Yes, can report similar issues. The output volume is already very low when it comes out of the DLNA stream (listening directly to the stream also has a really low volume).

If I overattenuate to 150% everywhere, I get a very bad quality but still rather insufficiently listenable audio stream.

Testing with parecord, I can find no significant issue with the volume itself (`parecord --file-format=wav -d alsa_output.pci-0000_00_1b.0.analog-stereo.monitor test.wav` is about the same volume as the result from `parecord --file-format=wav -d gmediarender_dlna.monitor test2.wav`), so I'm guessing the encoding with `sox` is broken somehow.

For reference, I get the following output from `ps` as to what's happening, in this case with my WAV encoder attempt:

towo     19338  0.1  0.1 355244 25764 pts/9    Sl+  23:20   0:01  |       \_ stream_server                                     
towo     19114  1.0  0.0 133992  5060 pts/9    S+   23:41   0:00  |       |   \_ parec --format=s16le -d gmediarender_dlna.monitor
towo     19115  0.0  0.0  27320  2260 pts/9    S+   23:41   0:00  |       |   \_ sox -t raw -b 16 -e signed -c 2 -r 44100 - -t wav -b 16 -e signed -c 2 -r 44100 -L -

**Comment by towo** [comment]
Hmm, encoding _in principle_ seems to work fine as well - grabbing the `oggenc` output from an ogg run, for example, gives a perfectly fine volume all round, as well.

**Comment by towo** [comment]
Disregard that, it seems to be a bug concerning what [gmrender-resurrect](https://github.com/hzeller/gmrender-resurrect) does with the stream in my case.

**Comment by Bergiu** [comment]
I have the same problem. I am using Kodi and it shows a connection to my Laptop but there is no sound and the visualisation stays flat.

**Comment by silentjay** [comment]
I'm also experiencing this issue. I have to have my speakers full blast just to get a barely listenable volume.   Have tried lots of different options when starting pulseaudio-dlna, doesn't make any difference. I'd say it loses about 70-80% of its volume.

Any ideas?

**Comment by silentjay** [comment]
Fixed it, connected my android phone to my chromecast and realised the chromecast has its own volume, which was set low. Once adjusted everything plays fine.

**Comment by masmu** [comment]
@silentjay Thanks for mentioning that. You can also control the Chromecasts volume via the Google "Home" App.

**Comment by rastapopougros** [comment]
Many months later, I have to say that this issue is absolutely not "closed" for me. :(

Always same prob with always same informations : Teac CR-H700, rhythmbox or any other thing with sound, I see my hi-fi in the config and ALL config is 100% in audio setting and in Pavu.

I still have sound in the hi-fi Teac. But miniminiminimini. And obsiously, if I put the sound louder and louder to ear, it's ugly noisy.

**Comment by rastapopougros** [comment]
About one year later, still not close for this Teac wifi hi-fi.

Still see it in the pavucontrol, still working connection between my PC and the hi-fi, but still with a super super low sound, with all at 100% everywhere.

**Comment by rastapopougros** [comment]
4 years after the creation, I still don't see why this issue is closed, because the problem is not solved at all.

All softwares sound config (in ubuntu, in rhythmbox, in firefox, etc) are at 100%. My Teac CR-H700 is still detected by Ubuntu when pulseaudio-dlna is working. But the sound is sent to my stereo with a mega low volume.

I know, I repeat, but absolutely all is still the same as when I created this issue.

Also I tested to send music on this same hifi from an Android phone, with bubbleupnp, and the sound is PERFECT without doing anything. And there isn't any additional volume control available on the phone app, it just works. So it's really the sound sent by pulseaudio-dlna which is like said by @silentjay  "about 70-80% loss".
